src/app.js

```diff
@@ -694,7 +707,9 @@ function run () {
     app: self.event,
     udapp: udapp.event
   }
-  var righthandPanel = new RighthandPanel(document.body, rhpAPI, rhpEvents, {}) // eslint-disable-line
+  var righthandPanel = new RighthandPanel(rhpAPI, rhpEvents, {}) // eslint-disable-line
+  self._view.rightpanel.replaceWith(righthandPanel.render())
```

Comment:
because adding `RighthandPanel` in the `.render()` function and not in the `.init()` call also requires ALL the dependencies of RighthandPanel to be already initialized ... this has a lot of ripple effects, so i would rather slowly go there to not again end up with a huge pull request :/